applications/SwimmingDEMApplication/custom_python/add_custom_constitutive_laws_to_python.cpp

```diff
@@ -134,6 +135,12 @@ void AddCustomConstitutiveLawsToPython(pybind11::module& m) {
         .def(py::init<Parameters&>())
         ;
 
+    py::class_<ChienDragLaw, ChienDragLaw::Pointer, BaseDragLawType>(m, "ChienDragLaw")
+        .def(py::init<>())
+        .def(py::init<Parameters&>())
```

Comment:
```suggestion
        .def(py::init<Parameters>())
```

`Parameters` should be passed by value